test/__snapshots__/browsers.test.ts.snap

```diff
@@ -1,5 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`connect flow create account 1`] = `"{\\"0xa98F414D87543ff275167c46A53AB987dBBF0c10@eip155:1\\":\\"821b662e859b69164df27549f4fba12727734a3ed853f992c17c2801e6e919a2\\"}"`;
+exports[`connect flow create account 1`] = `"{\\"0xa98F414D87543ff275167c46A53AB987dBBF0c10@eip155:1\\":\\"a7291389a402825fd7f07f9902faf24c21134c26d35a0d4618702c0fcd7c556e\\"}"`;
 
-exports[`connect flow get DID provider 1`] = `"{\\"0xa98F414D87543ff275167c46A53AB987dBBF0c10@eip155:1\\":\\"821b662e859b69164df27549f4fba12727734a3ed853f992c17c2801e6e919a2\\"}"`;
+exports[`connect flow get DID provider 1`] = `"{\\"0xa98F414D87543ff275167c46A53AB987dBBF0c10@eip155:1\\":\\"a7291389a402825fd7f07f9902faf24c21134c26d35a0d4618702c0fcd7c556e\\"}"`;
```

Comment:
Hopefully my suggested change should fix this.

Reply:
Yes seems to be it, thanks!

Reply:
Sweet 🎉